src/topics/group-generator/group-generator.api.test.ts

```diff
@@ -0,0 +1,10 @@
+import request from "supertest";
+import app from "../../api/app";
+
+describe("test groups generator api", () => {
+  test("Should get not empty group generator", async () => {
+    const response = await request(app).get(`/group-generators`);
```

Comment:
You could add a jest mock just before with one or 2 group generators and verify it in the API response instead on just the length